### DISCUSSION TITLE
Install Issue server.py

### DISCUSSION THREAD

**Discussion by unconame** [oobabooga/text-generation-webui#5065] [General]
When I run wizardlm it comes up with this error -   File "C:\TCHT\text-generation-webui\server.py", line 4, in <module>                                                                                                       from modules import shared                                                                                                                                          ImportError: cannot import name 'shared' from 'modules' (unknown location)    . I have spent 2 days trying to find out how to fix this issue.  any suggestions? thanks

**Comment by sebastianlutter** [comment]
I'm pretty sure you somehow deleted the modules folder content or you overlay it with something. I had the same error when I mounted my models folder over the modules folder by accident.
